dataeng/resources/opsgenie-enable-heartbeat.sh

```diff
@@ -0,0 +1,24 @@
+# Enables an Opsgenie Heartbeat, while also setting the expected duration.
```

Comment:
Doesn't this only enable the heartbeat?  Don't you still have to ping it?

Reply:
There will be no `/ping` issued. Instead, we'll simply use the `/disable` as the "ping" when the job finishes.

Reply:
I assume you've tested this or considered whether it would basically work, and I haven't read the docs so I'll approve, but I'm still curious: is the enable/PATCH call is an implicit ping?

Reply:
@pwnage101 Turns out that I *did* need to ping it once after enabling it. So the PATCH call was *not* an implicit ping - fixed now.